### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by mahermer88**
I know it is a lot of change, but I had to reformat all code.

### CODE REVIEW

File: public/style.css [hunk 1]
```diff
@@ -7,108 +7,167 @@
...
-  background-color: cadetblue;
+  padding: 0;
+  margin: 0;
   font-family: 'Concert One', cursive !important;
   font-size: 1.5rem !important;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: linear-gradient();
```
Comment (martijnarts) [inline public/style.css]: What's this for?

File: src/listeners/onload.js [hunk 2]
```diff
@@ -1,17 +1,15 @@
-import { displayQuestion } from '../views/questions-list.js';
-import { startGame } from '../handlers/startGame.js';
-import { setNextQuestion } from '../handlers/setNextQuestion.js';
-import { data} from '../data.js';
+import { startGame } from '../handlers/start-game.js';
+import { setNextQuestion } from '../handlers/set-next-question.js';
+import { questionToDelete } from '../handlers/delete-Question.js';
+import { questionToAdd } from '../handlers/add-Question.js';
```
Comment (martijnarts) [inline src/listeners/onload.js]: `delete-question.js` and `add-question.js` should have lowercase Q's here

File: src/views/render-question.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  //Adding the text of each question
+  const questionText = document.getElementById('question');
+  questionText.innerHTML = question.text;
+
+  //Adding the answers of each question:
+  const answers = question.answers;
+  for (const [key, value] of Object.entries(answers)) {
+    const button = document.createElement('button'); // Creating the button of each answer
+    button.classList.add('btn');
+    button.innerText = value; // add the test of answer
```
Comment (martijnarts) [inline src/views/render-question.js]: "test" should probably be "text"

File: src/views/render-question.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+  //Adding the link of resources of each question to the get hint part
+  const hintBox = document.getElementById('hint-box');
+  hintBox.classList.remove('hide');
+  for (const link of question.links) {
+    const href = link.href; // get the link of resource of each question
+    const hint = document.createElement('a');
+    hint.innerHTML = 'Source'; // insert word hint
+    hint.href = href; // insert the link
+    hint.target = '_blank'; // make the link open in new tab
```
Comment (martijnarts) [inline src/views/render-question.js]: The code here is obvious enough that it shouldn't need comments

**Review by martijnarts — CHANGES_REQUESTED** [review]
Tiny comments

File: public/style.css (hunk 1 above)
Comment (mahermer88) [reply public/style.css]: it is a mistake

File: src/handlers/select-answer.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ import { renderScore } from '../handlers/render-score.js';
 
 export function selectAnswer(event) {
   const selectedButton = event.target;
-
+  selectedButton.style.borderColor = 'gold'
```
Comment (martijnarts) [inline src/handlers/select-answer.js:7]: Missing a semicolon

**Review by martijnarts — APPROVED** [review]
Looks good!

**Pull request merged**